### PR TITLE
Feature/browserify ify

### PR DESCRIPTION
**Pull request by jmcarp**
Solves the same problem as #97, but moves more dependencies (jquery and typeahead) into browserify and out of source control / CDNs. I tested this manually and everything looks fine to me, but could use review from @msecret or @noahmanger.

### CODE REVIEW

File: static/js/modules/accordion.js [hunk 1]
```diff
@@ -119,4 +121,4 @@ var accordion = {
   }
 };
 
-module.exports = accordion;
\ No newline at end of file
+module.exports = accordion;
```
Comment (msecret) [inline static/js/modules/accordion.js]: Why was the export removed?

Comment (jmcarp) [inline static/js/modules/accordion.js]: I'm a little confused about this one, but it looks like GitHub is displaying the diff wrong. I didn't delete that line, but it's showing up as deleted anyway. Check out the modified file at https://github.com/jmcarp/openFEC-web-app/blob/feature/browserify-ify/static/js/modules/accordion.js. It has that line intact.

File: static/js/modules/typeahead.js [hunk 2]
```diff
@@ -1,5 +1,11 @@
 'use strict';
+
+/* global require, Bloodhound */
+
+var $ = require('jquery');
+require('typeahead.js/dist/typeahead.bundle.js');
```
Comment (msecret) [inline static/js/modules/typeahead.js]: Just `require('typeahead');` doesn't work? I was reading in various issues that it should potentially.

Comment (jmcarp) [inline static/js/modules/typeahead.js]: You're right--will change this.

**Pull request merged**